bears/perl/PerlCriticBear.py

```diff
@@ -1,36 +1,36 @@
-import re
+import shutil
 
-from coalib.bearlib.abstractions.Lint import Lint, escape_path_argument
-from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.abstractions.Linter import linter
 from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
 
-class PerlCriticBear(LocalBear, Lint):
-    executable = 'perlcritic'
-    output_regex = re.compile(
-            r'(?P<line>\d+)\|(?P<column>\d+)\|(?P<severity>\d+)\|'
-            r'(?P<origin>.*?)\|(?P<message>.*)')
-    severity_map = {
-        "1": RESULT_SEVERITY.MAJOR,
-        "2": RESULT_SEVERITY.MAJOR,
-        "3": RESULT_SEVERITY.NORMAL,
-        "4": RESULT_SEVERITY.NORMAL,
-        "5": RESULT_SEVERITY.INFO}
-    LANGUAGES = "Perl"
+@linter(executable=shutil.which("perlcritic"),
+        output_format='regex',
+        output_regex=r'(?P<message>.+) at '
+                     r'line (?P<line>\d+), '
+                     r'column (?P<column>\d+). '
```

Comment:
example output:
``Code before strictures are enabled at line 1, column 1.  See page 429 of PBP.  (Severity: 5)``